db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -172,6 +172,11 @@ abstract class DbCommonsColumnMappers {
     MappedColumnType
       .base[SatoshisPerByte, Long](_.toLong, SatoshisPerByte.fromLong)
   }
+
+  implicit val addressTagMapper: BaseColumnType[AddressTag] = {
+    MappedColumnType
+      .base[AddressTag, String](_.name, tag => CustomAddressTag(tag, ""))
```

Comment:
should probably give this constructor a special name rather than using `apply` as we generally don't want people doing this by accident